_maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm

```diff
@@ -43,9 +46,15 @@
 /area/ruin/powered/seedvault)
 "aj" = (
 /obj/structure/table/wood,
-/obj/item/gun/energy/floragun,
-/obj/item/gun/energy/floragun,
-/obj/item/gun/energy/floragun,
+/obj/item/gun/energy/floragun{
+	pixel_z = 6
+	},
+/obj/item/gun/energy/floragun{
+	pixel_y = 4
+	},
+/obj/item/gun/energy/floragun{
```

Comment:
I've notcied that you've given lifebringers 3 of each of these kinds of tools... but aren't there 4 lifebringer spawners?

Reply:
If you read closer, you'll see that there are 4 total.